src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx

```diff
@@ -0,0 +1,168 @@
+import { Flex, Grid, GridItem, Text } from "@chakra-ui/react";
+import { Color, GlobeIcon, InstagramIcon, MetricLabel, TwitterIcon } from "@dex-ui-components";
+import { useOutletContext } from "react-router-dom";
+import { GovernanceDAODetailsContext } from "./types";
+
+export function GovernanceDAODashboardOverview() {
+  const { dao, totalAssetValue, tokenCount } = useOutletContext<GovernanceDAODetailsContext>();
+  return (
+    <Flex gap="8" direction="column">
+      <Flex gap="4" direction="column">
+        <Text textStyle="h4 medium">Overview</Text>
+        <Grid templateColumns="repeat(2, 1fr)" gap={2}>
+          <GridItem>
+            <Flex
+              direction="column"
+              width="100%"
+              height="100%"
+              bg={Color.White_02}
+              justifyContent="space-between"
+              border={`1px solid ${Color.Neutral._200}`}
+              borderRadius="4px"
+              padding="1.5rem"
+              gap="8"
+            >
+              <Text textStyle="p medium semibold">Assets</Text>
+              <Flex direction="row">
+                <Flex flex={1}>
+                  <MetricLabel
+                    label="TOTAL ASSETS"
+                    labelTextColor={Color.Neutral._500}
+                    labelTextStyle="p xsmall medium"
+                    labelOpacity="1.0"
+                    value={totalAssetValue ?? 0}
+                    valueStyle="p large semibold"
+                    valueTextColor={Color.Grey_Blue._800}
+                    valueUnitSymbol="USD"
+                    valueUnitSymbolColor={Color.Grey_Blue._800}
+                  />
+                </Flex>
+                <Flex flex={1}>
+                  <MetricLabel
+                    label="TOKENS"
+                    labelTextColor={Color.Neutral._500}
+                    labelTextStyle="p xsmall medium"
+                    labelOpacity="1.0"
+                    value={tokenCount ?? 0}
+                    valueStyle="p large semibold"
+                    valueTextColor={Color.Grey_Blue._800}
+                  />
+                </Flex>
+              </Flex>
+            </Flex>
+          </GridItem>
+          <GridItem>
+            <Flex
+              direction="column"
+              width="100%"
+              height="100%"
+              bg={Color.White_02}
+              border={`1px solid ${Color.Neutral._200}`}
+              borderRadius="4px"
+              padding="1.5rem"
+              gap="8"
+            >
+              <Text textStyle="p medium semibold">Governance</Text>
+              <Flex direction="row" justifyContent="space-between">
+                <MetricLabel
+                  label="QUORUM"
+                  labelTextColor={Color.Neutral._500}
+                  labelTextStyle="p xsmall medium"
+                  labelOpacity="1.0"
+                  value={dao.quorumThreshold}
```

Comment:
Threshold is showing up as `undefined`:
<img width="389" alt="Screen Shot 2023-05-28 at 9 49 23 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/a509fb6d-7467-4278-ba34-693c7d4b410d">

Reply:
I will update settings page with NFT DAO Dashboard (https://rally1.rallydev.com/#/641170237735d/iterationstatus?detail=%2Fuserstory%2F699500683855). as Settings for Governance and NFT DAOs are different from MultiSig DAO. 

design: https://www.figma.com/file/moKTRbk1JAqXAuh5n759u2/Hedera-DEX-UI-designs?type=design&node-id=2254-92604&t=xxBjn45cqQkR89SV-0